build-system/tasks/e2e/expect.js

```diff
@@ -236,8 +238,40 @@ function overwriteUnsupported(_super) {
   };
 }
 
+function installBrowserAssertions(_networkLogger) {
+  networkLogger = _networkLogger;
+  chai.use(installBrowserWrappers);
+}
+
+function installBrowserWrappers(chai, utils) {
+  const {Assertion} = chai;
+
+  const assertAnySentRequests = function(url) {
+    chai
+      .expect(networkLogger.getSentRequests(url))
+      .to.eventually.not.have.length(0)
+      .then(() => {
+        return true;
+      })
+      .catch(() => {
+        return false;
+      });
+  };
+
+  // Assert that a request with a testUrl was sent
+  // Example usage: await expect(testUrl).to.be.sent;
+  utils.addProperty(Assertion.prototype, 'sent', function() {
+    this.assert(
+      assertAnySentRequests(this._obj),
+      'expected #{this} to be sent',
+      'expected #{this} to not be sent'
```

Comment:
Grammar nit: I think Chai's assertion chaining (https://www.chaijs.com/api/bdd) should automatically allow you to use this version:
```js
await expect(testUrl).to.have.been.sent;
```

Reply:
SG, both work!